flask_monitoringdashboard/test/db/test_count.py

```diff
@@ -1,6 +1,6 @@
 """
     This file contains all unit tests that count a number of results in the database. (Corresponding to the file:
-    'dashboard/database/count.py')
+    'fmd_dashboard/database/count.py')
```

Comment:
The refactoring affected some of the comments and the paths don't make sense now. I'll fix this.